style.css

```diff
@@ -0,0 +1,1000 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
```

Comment:
- [ ] Your mobile version is well settled, smile however, you should make sure the content does not overflow the size of your screen, which should be  `375`px for a  Minimum mobile screen size 

![Screenshot from 2023-01-19 22-05-12](https://user-images.githubusercontent.com/80458199/213549142-5d60d97f-5f4b-4cd4-aa33-23288e962ee7.png)

Reply:
as highlighted by the previous reviewers, your page still has the bugs they pointed out. I suggest you concentrate on finishing the mobile version first, in the next milestone, that is when the desktop version comes into play.
The screenshot below shows your page on 375px width screen
<img width="441" alt="Screenshot 2023-01-20 at 14 29 20" src="https://user-images.githubusercontent.com/48546992/213685726-b2d18b7f-8ead-4513-b025-446af1f5fbb3.png"> 

This is how it should appear.
<img width="373" alt="Screenshot 2023-01-20 at 14 35 15" src="https://user-images.githubusercontent.com/48546992/213686239-65803483-1dba-4b54-aaaf-e493bfec298d.png">